src/vs/server/node/webClientServer.ts

```diff
@@ -183,12 +226,23 @@ export class WebClientServer {
 			accessToken: this._environmentService.args['github-auth'],
 			scopes: [['user:email'], ['repo']]
 		} : undefined;
+		const resourceUrlTemplate = this._productService.extensionsGallery ? URI.parse(this._productService.extensionsGallery.resourceUrlTemplate) : undefined;
```

Comment:
Is the idea of `resourceUrlTemplate` to be a prefix that paths are being concatenated to?

Reply:
Yes, for eg:

https://github.com/microsoft/vscode/blob/83d42f8877a72133dc76b151195a98d21ab36717/src/vs/workbench/services/extensionManagement/browser/webExtensionsScannerService.ts#L401